package/gluon-web-wifi-config/i18n/fr.po

```diff
@@ -46,3 +46,9 @@ msgstr ""
 "<br /><br />Ici vous pouvez aussi configurer la puissance d'émmission se votre Wi-Fi. "
 "Prenez note que les valeurs fournies pour la puissance de transmission prennent "
 "en compte les gains fournis par l'antenne, et que ces valeurs ne sont pas toujours disponibles ou exactes."
+
+msgid "Outdoor installation"
+msgstr "Installation extérieure"
+
+msgid "HT Mode"
+msgstr "Mode HT"
```

Comment:
This translation section should be in a file in an independent package-specific location.